packages/venia-concept/src/ThemedCss/frenchButton.css

```diff
@@ -0,0 +1,76 @@
+.root {
+    color: red;
+    border-color: red;
+}
```

Comment:
My intent was just for this to be the css that was needed. Should be possible to provide _just_ those overrides without having to know all the rest of the css classes.